### DISCUSSION TITLE
v4.9 release and option for back porting a fix

### DISCUSSION THREAD

**Discussion by Tanish0019** [JSQLParser/JSqlParser#1962] [General]
Hello, thanks a lot for maintaining this project.

I am currently using version v3.1 of this project and was running into this issue #1737. My PR for the fix has been merged to master #1961 

I had 2 questions:
1. Can you let me know by when we can expect the next version which will contain the fix? 
2. Will it be possible to back port the fix to v3.x and do new release for that as well?

Thanks!

**Comment by Tanish0019** [comment]
Hi, @manticore-projects would really appreciate if you could take a look at these questions. Thanks you!

**Comment by manticore-projects** [comment]
As a volunteer project we do not have any release policy and schedule.
In fact I see JSQLParser as a rolling release where every accepted commit is the latest greatest version. You can use the Snapshot Artifacts from SonaType.

This can only change, when a paying sponsor shows up.
